Sources/BraintreeCore/Public/BraintreeCore/BTLogger.h

```diff
@@ -3,6 +3,7 @@
 /**
  Braintree SDK Logging Levels
 */
+// Usage of this logger seems sparse
```

Comment:
I think there are places where having logs is helpful to merchants (for example:[ in BTAPIClient when their auth credentials are invalid](https://github.com/braintree/braintree_ios/blob/a234cdd0e070590b150c09f7d310f27198d6237e/Sources/BraintreeCore/BTAPIClient.m#L71)).

I do think we could consolidate some of these log levels, though. I think this doesn't need to be done prior to the v5 GA release.

Reply:
+1